modules/virt-creating-configmap.adoc

```diff
@@ -0,0 +1,264 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+[id='virt-creating-configmap_{context}']
+= Creating a custom ConfigMap for importing a virtual machine
+
+You can check whether the operating system of the Red Hat Virtualization (RHV) virtual machine that you are importing maps to an {VirtProductName} common template.
+
+If the RHV VM's operating system does not have a corresponding {VirtProductName} template, you must create a custom ConfigMap to map the VM's operating system to a common {VirtProductName} template. The template is used to create the new virtual machine in {VirtProductName}.
+
+[NOTE]
+====
+In the current release, you must create a Windows template and a custom ConfigMap to import a RHV VM with a Windows operating system. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1856654[BZ1856654] for details.
+====
+
+.Procedure
+
+. In a web browser, identify your RHV VM's operating system by navigating to `\http://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` as in the following example:
++
+[source,xml]
+----
+<os>
+...
+<type>windows_2016x64</type>
+</os>
+----
```

Comment:
The command dumps a long XML file. Providing these tags enables the user to search for the tag that identifies the operating system. Otherwise it's a lot of reading....